test/k8sT/Policies.go

```diff
@@ -741,6 +742,12 @@ var _ = Describe("K8sValidatedPolicyTestAcrossNamespaces", func() {
 	})
 
 	AfterEach(func() {
+		for _, resource := range resources {
+			resourcePath := kubectl.ManifestGet(resource)
+			res := kubectl.Delete(resourcePath)
+			res.ExpectSuccess()
```

Comment:
I think that the ExpectSucess should not be called in AfterEach, if something fails here the following lines will not work. Happy if you decided to keep but I do not really like, assertions are for tests and JustAfterEach IMHO.

Reply:
If you want to keep this ExpectSucess should have an assert message.

Reply:
I like this paradigm, makes sense to me.

Reply:
Fixed.